### DISCUSSION TITLE
Is it possible to do object composition

### DISCUSSION THREAD

**Discussion by FranciscoTorrinha** [rust-skia/rust-skia#958] [Q&A]
Hey guys,
I have multiple objects (TextBlobs) and I want to apply a series of transformations on them and they must behave as one. 
Is it all possible to create a custom "object" that is composed by the TextBlobs and apply the transformations to that custom "object"?

Thanks in advance,
Francisco

**Comment by Joaopmorais** [comment]
No,i don't think so. I think  you need to make the calculation for each one.
